modules/demo_account/actions.tf

```diff
@@ -0,0 +1,29 @@
+resource "opslevel_webhook_action" "security_scan" {
+  name        = "Security Scan"
+  description = "Runs a security scan for the service"
+  url         = "https://app.opslevel.com/upload/documents/sbom/{{service.id}}"
+  method      = "POST"
+  headers = {
+    content-type  = "application/json"
+    authorization = "Bearer ${var.account_token}"
+  }
+  payload = local.sbom_minimal
```

Comment:
we need to craft sbom data to support the flow of this action going form "broken" to "fixed".

First we need to craft some checks which "detect the problems"